app/controllers/offices_controller.rb

```diff
@@ -42,8 +42,10 @@ def update
     office.assign_attributes(office_params)
     authorize office
 
-    if office.save && manager_setup.in_progress?
-      redirect_to out_of_the_box_redirect
+    if office.save
```

Comment:
It's now being handled in the update_redirect_path here: https://github.com/ministryofjustice/fr-staffapp/pull/549/files#diff-5ab24263464befc902ee5117ae8011beR72